package.json

```diff
@@ -6,9 +6,13 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "lint": "eslint src __tests__ __mocks__",
     "lint:fix": "eslint --fix src __tests__ __mocks__",
-    "test": "flow && jest",
+    "test": "flow && jest __tests__",
```

Comment:
Why exactly was the `__test__` added?

Reply:
Without the __test__ parameter it would try and run the __e2e__ folder as well which would trigger the detox tests and would fail